### DISCUSSION TITLE
How to reset a single field?

### DISCUSSION THREAD

**Discussion by Jsramosra** [react-hook-form/react-hook-form#4332] [Q&A]
I'm trying to reset a single field referencing the object in the reset({MY_VARIABLE_NAME: ""}); but it kinda didn't work 😅, is there a way to do this?

**Comment by bluebill1049** [comment]
setValue if you want to update a single field

**Reply by Jsramosra** [reply]
Can you give me an example Bill?

**Reply by bluebill1049** [reply]
`setValue('fieldName', 'yourValue here')`

https://react-hook-form.com/api#setValue
